schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml

```diff
@@ -170,6 +170,11 @@
       <summary>TODO</summary>
       <description>TODO</description>
     </key>
+    <key type="b" name="show-notification-badges">
+      <default>false</default>
```

Comment:
This should be `true` by default.

Reply:
You are right, it should behave like the stock version of dash-to-dock. I somehow thought it was
on true... Thanks for pointing this out